src/base/static/js/routes.js

```diff
@@ -138,7 +162,10 @@ Shareabouts.Util = Util;
       if (Backbone.history.getFragment() === '') {
         startPageConfig = S.Util.findPageConfig(options.pagesConfig, {start_page: true});
 
-        if (startPageConfig && startPageConfig.slug) {
+        if (startPageConfig 
+          && startPageConfig.slug
+          // don't route to the start page on small screens
```

Comment:
I think only screens widths above `show_above_width` or 960px will run the `this.navigate` line, right?